spec/lucky/cookies/cookie_jar_spec.cr

```diff
@@ -117,7 +126,7 @@ describe Lucky::CookieJar do
     it "raises an error if the cookie is > 4096 bytes" do
       expect_raises(Lucky::CookieOverflowError) do
         jar = Lucky::CookieJar.empty_jar
-        jar.set_raw(:overflow, "x" * (4097 - 27)) # "overflow=x...x; path=/; HttpOnly",
+        jar.set_raw(:overflow, "x" * 4097) # "overflow=x...x; HttpOnly",
```

Comment:
I'm not sure why there was maths here, but it wasn't throwing an exception, so I just made it the raw number because `"x" * 4097` is `4097 bytes`